yarn-project/aztec-cli/src/index.ts

```diff
@@ -143,10 +143,9 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
   program
     .command('deploy')
     .description('Deploys a compiled Noir contract to Aztec.')
-    .option(
+    .requiredOption(
```

Comment:
Are `--salt` and `--public-key` also a required options, here?

Reply:
Hello, yes in this case it works without them just fine.